### DISCUSSION TITLE
CTkComboBox bind <<ComboboxSelected>>

### DISCUSSION THREAD

**Discussion by staceellis** [TomSchimansky/CustomTkinter#321] [General]
Hello, is it possible to create an "ComboboxSelected" event binding to a CTkComboBox like is possible with the ttkComboBox?
I have attempted the code below which doesn't seem to work. I have also tried with "Button-1" but this also doesn't seem to work.

```
self.str_variable = StringVar()

def combobox_event(self):
    print(self.str_variable.get())

self.combobox = customtkinter.CTkComboBox(master=self,
                                          values=self.list,
                                          variable=self.str_variable)
self.combobox.grid(row=0, column=0, pady=(0, 10), padx=20)
self.combobox.bind("<<ComboboxSelected>>", self.combobox_event)
```

Any advice would be much appreciated.
Thank you.

**Comment by LeadwoodSystems** [comment]
Is there a solution for the above? 
I have encountered the same problem. 

I attempted to update Combobox2 values based on the selected value from ComboBox1.
To no avail.

**Reply by LeadwoodSystems** [reply]
Figured it out: 

````
self.combobox = customtkinter.CTkComboBox(master=self,
                                          values=self.list,
                                          command = callback_func,

def callback_func(self):
        self.combobox.set("")
        self.combobox.configure(values=new_values_list))
````

add the `command` param instead of `bind`
in the function use first reset the combobox using `set` (if you want) 
`self.combobox.set("")`
then `configure`
`self.combobox.configure(values=new_values_list))`
